### DISCUSSION TITLE
Ring Notifications for Google Home

### DISCUSSION THREAD

**Discussion by Luke1505** [koush/scrypted#930] [Q&A]
Hi There,
I've set up my Scrypted Raspberry Pi and came to the issue that I wanted to Connect my Ring Doorbell with Google Home which works well. Still, I am only able to see the Camera Stream and I don't get any notification or whatever that the doorbell rang. 
Do you know if there is a way to fix that ? Do I need to change something?

Kind Regards 
Luke

**Comment by scho0ck** [comment]
Did you ever find an answer? Have the same issue.
